tools/cockpit-sysusers.conf

```diff
@@ -0,0 +1,2 @@
+u cockpit-ws - "User for cockpit web service" -
+u cockpit-wsinstance - "User for cockpit-ws instances" -
```

Comment:
This "static" duplication is rather ugly. It's also not really the result of substituting src/systemd/cockpit-sysusers.conf.in, as it misses the group and the comments. To be honest, let's just ditch the configurability of this, i.e. drop the fouru `--with-cockpit-{,ws-instance-}{user,group}` configure options.  The default and packaging is the same in all OSes anyway, and there's value in making this consistent between all OSes (one of the big advancements that systemd brought to the fractured Linux world). This will simplify the mechanics here a lot.

Reply:
Added a commit to remove the configurability.

Reply:
This can now be dropped -- it's exactly the same as src/systemd/sysusers.d/cockpit-ws.conf.